docs/migration-guide.md

```diff
@@ -15,6 +15,36 @@ native tools and utilities for which they were originally created. The biggest b
 is the necessity to use external tools *alongside* Neutrino now, i.e. webpack, ESLint, Jest, Karma,
 and others' native CLIs will be used in tandem with Neutrino.
 
+Additionally, the Neutrino API no longer accepts any middleware format other than functions.
+The `.neutrinorc.js` file is an exception, which still supports an object-based definition
+for setting Neutrino options. There is now a new package to assist in migration changes to core
+Neutrino middleware which are now middleware factories, since packages can no longer be required
+for you:
+
+```bash
+npx @neutrinojs/migrate rc
```

Comment:
If I use this command as stated I get a "Skipping path C:\Users\Ed\src\treeherder\rc which does not exist.".
I'm presuming this should now be `npx @neutrinojs/migrate` with no additional arguments?